### ISSUE TITLE
Unable to reset password

### ISSUE THREAD

**Issue by isaacgr** [firefly-iii/firefly-iii#6884]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Similar issue to previous reports about not being able to reset their password. 

I did the steps listed in #1611. This worked to get me a reset link, however when I submit my new password I get 

> Error! There were some problems with your input.
> This password reset token is invalid.

On the page

There doesn't appear to be any output in the logs as to why this is.

### Expected behaviour

The password reset link should work.

### Debug information

I cant access this.

**Comment by JC5** [comment]
I'm not sure if instructions from 2018 are still valid though 😉 

I can't seem to replicate this, sorry. You're saying the email arrives just fine but then the link doesn't work?

**Comment by isaacgr** [comment]
Not sure if its like its a date setting thing? My server and PC are in EST. The firefly app is in CET and the mysql db is in UTC. Not sure what the expiry time is on the token. Doing the conversion to EST the server and db are the same time. The time that gets inserted into the db is the CET time of the firefly server which makes sense.

**Comment by isaacgr** [comment]
> I'm not sure if instructions from 2018 are still valid though 😉
> 
> I can't seem to replicate this, sorry. You're saying the email arrives just fine but then the link doesn't work?

Yes thats correct. The link allows me to get to the reset page, but then says its invalid.

![image](https://user-images.githubusercontent.com/20640588/212890222-ea5f9cde-83b6-4fe6-aeac-896eb16a70e9.png)
